<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtocolLabelDialog</class>
 <widget class="QDialog" name="ProtocolLabelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protocol label</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>External program:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Label:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QSpinBox" name="spinBoxLabelStart">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QSpinBox" name="spinBoxLabelEnd">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QComboBox" name="comboBoxName">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QComboBox" name="comboBox_3">
     <item>
      <property name="text">
       <string>Add</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Subtract</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Adjust value:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>End index:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Start index:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Value:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5" colspan="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="11" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="11" column="6">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="3">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="4" column="1" colspan="6">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>Get live during simulation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="QRadioButton" name="radioButton_4">
     <property name="text">
      <string>Constant value</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="6">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>Protocol label</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="6">
    <widget class="QRadioButton" name="radioButton_5">
     <property name="text">
      <string>External program</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="6">
    <widget class="QRadioButton" name="radioButton_3">
     <property name="text">
      <string>Random value</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="7">
    <widget class="QPushButton" name="btnSaveAndClose">
     <property name="text">
      <string>Save and Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
